requirements.txt

```diff
@@ -14,3 +14,4 @@ Flask-Script==2.0.5
 workdays==1.4
 unicodecsv==0.14.1
 cryptography==1.6
+git+https://github.com/alphagov/notifications-python-client.git@4.0.0#egg=notifications-python-client==4.0.0
```

Comment:
They don't support 2.* yet so it's not available on `pip2` :(
@leohemsted is looking into/ floating the idea of 2.7 support soon apparently